ndonnx/_funcs.py

```diff
@@ -596,128 +578,46 @@ def numeric_like(x):
 
 
 def broadcast_to(x, shape):
-    x = asarray(x)
-    if (out := x.dtype._ops.broadcast_to(x, shape)) is not NotImplemented:
-        return out
-    shape = asarray(shape, dtype=dtypes.int64)._core()
-    return x._transmute(lambda corearray: opx.expand(corearray, shape))
+    return x.dtype._ops.broadcast_to(x, shape)
 
 
 # TODO: onnxruntime doesn't work for 2 empty arrays of integer type
+# TODO: what is the appropriate strategy to dispatch? (iterate over the inputs and keep trying is reasonable but it can
```

Comment:
Duplicate?

Reply:
The TODO? I think it's different. I'm unsure how to make `concat` custom dtype friendly due to the unsafeness of naively dispatching in the order of the arguments. Happy to take any suggestions :).

Reply:
Yes, the TODO. Isn't this TODO referring to the exact same thing as the one in line 555?